rails-settings-cached.gemspec

```diff
@@ -32,4 +32,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "simplecov"
   s.add_development_dependency "minitest"
   s.add_development_dependency "codecov"
+  s.add_development_dependency "byebug"
```

Comment:
Move this line into Gemfile

Reply:
Remove this dependency.

Reply:
@huacnlee this is just a development dependency. it helps dev to debug effectively when improving the gem, no?